pageserver/src/page_cache.rs

```diff
@@ -87,6 +85,14 @@ pub fn get() -> &'static PageCache {
 pub const PAGE_SZ: usize = postgres_ffi::BLCKSZ as usize;
 const MAX_USAGE_COUNT: u8 = 5;
 
+static NEXT_ID: AtomicU64 = AtomicU64::new(1);
```

Comment:
A brief comment to explain what this whole file id thing is would be good.

Reply:
Why wasn't this needed before, or what handled it? Is this a move? It's a move from `block_io.rs`.